.changeset/four-peaches-attack.md

```diff
@@ -0,0 +1,8 @@
+---
+'playroom': patch
+---
+
+Disable snippet previews while the snippets panel is closing.
+
+Previously, snippet previews could be triggered while the snippet panel was closing, causing preview frames to enter an invalid state.
+Previewing a snippet will now only work when the snippet panel is open.
```

Comment:
This feels a bit too much like implementation detail to me. I'd just mention the bug and say it's fixed.

```suggestion
Fixes a bug that was causing duplicate snippet previews and broken preview updates when a snippet was clicked and the mouse was moved while the snippets panel was closing
```

It's a bit of a long sentence, but I think it gets the point across.